Library/Formula/bison.rb

```diff
@@ -14,6 +14,8 @@ class Bison < Formula
 
   keg_only :provided_by_osx, "Some formulae require a newer version of bison."
 
+  depends_on "homebrew/dupes/m4" => :build if OS.linux?
```

Comment:
I can never get it right. When is it OK to use `unless OS.mac?` and when `if OS.linux?`?